checkClock/app/javascript/packs/formCreateBranch.js

```diff
@@ -0,0 +1,84 @@
+import Swal from 'sweetalert2';
+
+let formCreate = document.querySelector("#aside-create");

```

Comment:
This could be refactored to follow a more scalable design for rails applications. But is good enough 😄